scripts/dts/edtlib.py

```diff
@@ -2164,6 +2646,12 @@ def _check_dt(dt):
 def _err(msg):
     raise EDTError(msg)
 
+_warnings = list()
```

Comment:
Better to have this as part of the `EDT` class, since it's possible to have multiple `EDT` instances. The library itself has no global state.

Reply:
Moved to EDT. Made some collateral moves necessary as _warn now needs EDT.